test/performance/README.md

```diff
@@ -2,23 +2,24 @@
 
 ## Configuring your cluster to run a benchmark
 
-1. Create a namespace or use an existing namespace. Each namespace can be
-   configured with a single benchmark.
+1. Create a namespace `perf-eventing` if not exists. To use a different
+   namespace, please replace all the namespaces in
+   all bash commands and yaml configuration files with your choice.
```

Comment:
do u mean bash scripts here or is commands intended?

Reply:
Command is commonly used to refer to one line script.